zone/npc.cpp

```diff
@@ -1311,10 +1355,10 @@ uint32 ZoneDatabase::AddNewNPCSpawnGroupCommand(
 
 	s2.zone         = zone;
 	s2.version      = instance_version;
-	s2.x            = n->GetX();
-	s2.y            = n->GetY();
-	s2.z            = n->GetZ();
-	s2.heading      = n->GetHeading();
+	s2.x            = c->GetX();
+	s2.y            = c->GetY();
+	s2.z            = c->GetZ();
+	s2.heading      = c->GetHeading();
```

Comment:
Same as above comment.

Reply:
Resolved and tested in new [commit](https://github.com/EQEmu/Server/pull/4311/commits/6524e60ba154786a0f7a5727e0717644fbd6a1d3)